### DISCUSSION TITLE
Is this a drop-in replacement for REST Client?

### DISCUSSION THREAD

**Discussion by YAMLcase** [AnWeber/vscode-httpyac#271] [General]
I recently discovered `httpyac` when looking for a stand-alone runner for my *.http files.  Some of the cool features in httpyac (@host variable for instance) weren't supported in Rest client so I eventually found this project.  It would be nice if I can just drop vscode-httpyac in place of Rest client and go about my day.  Is this possible?

**Comment by AnWeber** [comment]
Not quite, as I deviate slightly from a few features (also see #266).

- Response access using JSON Path or XPath is not supported support (use Javascript access with `response.parsedBody` instead)
- reference to other responses use object with request and response. I directly reference the response
- no support for `$dotenv`. just use variables directly
- no explicit Azure AD (`$aadV2Token`) support but should be replaced with OAuth2 Provider
- GraphQL does not filter the special GraphQL Header, Hader is not needed in httpyac

I think that should be all the missing features, but no guarantee. You could support vscode.rest-client 100%, but that wasn't important to me after a certain point.
